docs/full/policy.md

```diff
@@ -0,0 +1,132 @@
+# Tanzu CLI policy
+
+This document defines the version policy, plugin/tanzu-cli compatibility policy, support policy, and deprecation policy of the Tanzu CLI.
+
+## Tanzu CLI Version policy
+
+Tanzu CLI and [Tanzu Plugin Runtime Library](https://github.com/vmware-tanzu/tanzu-plugin-runtime) use a three-part version scheme: `<major>.<minor>.<patch>`.
+Changes to each of these versions have a very specific meaning:
+
+- major: indicates significant feature changes and possible breaking changes
+- minor: indicates functional changes (primary feature delivery)
+- patch: indicates minimal risk changes (critical security vulnerabilities and bug fixes)
+
+Given this scheme, the primary vehicle for changes is minor releases.  The deprecation policy assumes that all modifications and removals happen as part of minor releases (possibly rolled up into a major release).
+
+This section specifies the version, support, and deprecation policies for Alpha, Beta, and GA releases.
+
+## Plugin/Tanzu-CLI Compatibility policy
+
+- Each Tanzu Core CLI release should be compatible with all the existing plugins at the time of release and also with plugins developed with any future patch of the existing minor releases of Tanzu Plugin Runtime.
+  - For example, the below table shows the compatible plugin version with different releases of Tanzu CLI.
```

Comment:
@anujc25 did you mean to include a table?

Reply:
Good catch. I was planning to but missed it. I will add it.